Notesnook.API/Models/SyncItem.cs

```diff
@@ -71,7 +71,7 @@ public string Cipher
         [DataMember(Name = "id")]
         [JsonPropertyName("id")]
         [MessagePack.Key("id")]
-        public string ItemId
+        public string? ItemId
```

Comment:
have to make these fields nullable because `InboxSyncItem` inherits from `SyncItem`, and `InboxSyncItem` is used as request type in the controller. Since `ItemId` and `UserId` aren't present, the api throws an error.

though making these fields nullable here doesn't feel right, should we remove the inheritance, or?

Reply:
I think it's alright. It's an accurate representation of the model.